onadata/apps/main/views.py

```diff
@@ -783,6 +793,13 @@ def edit(request, username, id_string):
                 and request.FILES.get("xls_template"):
             template_upload_name = request.POST.get("template_upload_name")
             external_url = request.POST.get("external_url")
+
```

Comment:
@noliveleger, this is fine, but I don't think we expose this `external_exports` functionality at all, and probably all traces of it should be removed from our slimmed-down version of KoBoCAT.